src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -55,12 +68,48 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
             body = needsParentheses(actualExpression) ? createParen(actualExpression) : actualExpression;
             suppressLeadingAndTrailingTrivia(body);
             copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            // Copy the trailing comments after the return statement
+            // Add them to our trailingCommentsHolder to use later
+            copyTrailingComments(returnStatement, trailingCommentsHolder, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+
+
+            // If there are trailing comments
+            if (trailingCommentsHolder && trailingCommentsHolder.emitNode && trailingCommentsHolder.emitNode.trailingComments) {
+
+                // First we format our comment
+                // ex. " trailing comment " -> "/* trailing comment */"
+                const formattedComment = formatSynthesizedComment(trailingCommentsHolder.emitNode.trailingComments[0]);
+
+                // Add one space of padding to the comment
+                const paddedComment = ` ${formattedComment}`;
+
+                // TODO - add some type of check to know if plus 1 is needed

```

Comment:
I haven't done this yet, but I probably should because we only need to account for an extra character (i.e. the `+ 1`) if there is a semicolon.